src/lib/request/fetch.ts

```diff
@@ -1,9 +1,3 @@
-import fs from 'fs';
-import matter from 'gray-matter';
-import path from 'path';
-
-const basePath = path.resolve(`./src/blogposts`)
```

Comment:
This was the culprit.

It seems that Vercel production deployments resolve this `path` differently.


I've removed this code entirely - removed the `gray-matter` dependency too!

Now I fetch all files using a more idiomatic svelte approach using `glob`. See L24